package.json

```diff
@@ -19,7 +19,9 @@
     "@types/config": "^3.3.0",
     "@types/jest": "^29.2.4",
     "@types/node": "18.11.15",
+    "@types/source-map-support": "^0.5.6",
```

Comment:
do we need these dependencies? If we aren't using them, we should not add. It will increase the execution memory consumption when we put this on automated pipelines.